rpcserver.go

```diff
@@ -1648,8 +1648,8 @@ func handleGenerate(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (i
 
 	blockHashes, err := s.server.cpuMiner.GenerateNBlocks(c.NumBlocks)
 	if err != nil {
-		return nil, rpcInternalError("Could not generate blocks",
-			"Configuration")
+		msg := fmt.Sprintf("Could not generate blocks, %v", err)
+		return nil, rpcInternalError(msg, "Configuration")
```

Comment:
The first parameter of `rpcInternalError` is precisely for the error, and, realistically, the context in the existing error is incorrect.  This really should be:

```Go
		return nil, rpcInternalError(err.Error(), "Could not generate blocks")
```